rtloader/three/three.cpp

```diff
@@ -840,6 +840,34 @@ bool Three::getAttrString(RtLoaderPyObject *obj, const char *attributeName, char
     return res;
 }
 
+bool Three::getAttrBool(RtLoaderPyObject *obj, const char *attributeName, bool &value) const
+{
+    if (obj == NULL) {
+        return false;
+    }
+
+    bool res = false;
+    PyObject *py_attr = NULL;
+    PyObject *py_obj = reinterpret_cast<PyObject *>(obj);
+
+    py_attr = PyObject_GetAttrString(py_obj, attributeName);
+    if (py_attr != NULL) {
+        int truth = PyObject_IsTrue(py_attr);
```

Comment:
I think `PyObject_IsTrue` is a bit too loose/lenient, and will accept values like HA_SUPPORTE = "foo" as truthy/valid, I think we want to prevent that.

What about checking if it's a bool with `PyBool_Check` ? and then check if the value is True